guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -37,7 +36,7 @@ The base recommended external database configuration adjustments are as follows:
 * work_mem: 4MB
 
 ----
-# vi /var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf
+# vi /var/lib/pgsql/data/postgresql.conf
```

Comment:
Below this it doesn't render well. I think the block is missing `options="nowrap" subs="verbatim,quotes"]`.

Reply:
If you're using attributes, use `[options="nowrap", subs="+quotes,verbatim,attributes"]`.

Reply:
is there a way to configure asciidoc to *always* use these settings for code blocks?

Reply:
I don't know, but it would be a nice to have for sure!